### PR TITLE
Add Caixabank icon

### LINKED ISSUE #12269: Request: CaixaBank
### Brand Name

CaixaBank

### Website

https://caixabank.es

### Popularity Metric

[2,499 on SimilarWeb](https://www.similarweb.com/website/caixabank.es)

### Official Resources for Icon and Color

- SVG: at website footer https://www.caixabank.es/deployedfiles/particulares/Estaticos/Imagenes/Geolocalizador_CXB.svg
- Color: `#007EAE` from logo

### Additional Comments

Include only the star and the two points.

### PR DESCRIPTION
**Pull request by 8bearings**
image

**Issue:** closes #12269

**Popularity metric:** Similarweb Global Rank [2,499](https://www.similarweb.com/website/caixabank.es)

### Checklist

- [  ✔] I updated the JSON data in `_data/simple-icons.json`
- [  ✔] I optimized the icon with SVGO or SVGOMG
- [  ✔] The SVG `viewbox` is `0 0 24 24`

### Description

- SVG: at website footer https://www.caixabank.es/deployedfiles/particulares/Estaticos/Imagenes/Geolocalizador_CXB.svg
- Color: `#007EAE` from logo

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2788,6 +2788,11 @@
             "hex": "C10C0C",
             "source": "https://en.wikipedia.org/wiki/File:Cairo_metro_logo2012.svg"
         },
+        {
+            "title": "CaixaBank",
+            "hex": "007EAE",
+            "source": "https://www.caixabank.es/deployedfiles/particulares/Estaticos/Imagenes/Geolocalizador_CXB.svg"
```
Comment (LitoMore) [inline _data/simple-icons.json:2794]: The site is down. I can't verify the source at this time.

Comment (8bearings) [reply _data/simple-icons.json:2794]: Looks like the site is back up

Comment (LitoMore) [reply _data/simple-icons.json:2794]: https://downforeveryoneorjustme.com/caixabank.es?www=1



I still cannot visit the site.

Comment (8bearings) [reply _data/simple-icons.json:2794]: Odd, I am still able to access the site just fine
Screenshot 2024-12-09 at 3 38 51 PM

Comment (LitoMore) [reply _data/simple-icons.json:2794]: I still can not open the site. I've tried many country IPs through the VPN, but no luck.

**Review by LitoMore — APPROVED** [review]
@8bearings Congrats on your first contribution to Simple Icons!

Thanks!

**Pull request merged**